.github/workflows/release-published.yml

```diff
@@ -538,4 +538,53 @@ jobs:
               -Dclassifiers=,sources,javadoc,
           
           done
-            
\ No newline at end of file
+            
+  publish_assets_to_internal_nexus:
+    if: ${{ inputs.dry_run == false }}
+    name: Publish OSS released assets to repo.liquibase.net
+    needs: [ setup ]
```

Comment:
Another thought. Should the actual release step depend on this step? If not, won't they run concurrently, and the release step would still overwrite the existing release artifacts?